Makefile

```
@@ -63,3 +64,24 @@ delete-dependencies-and-resources:
 
 delete-operator:
 	oc delete -f deploy/operator.yaml
+
+delete-metering-report-resources:
+	oc delete report cm-openshift-node-labels
+	oc delete report cm-openshift-node-labels-lookback
+	oc delete report cm-openshift-persistentvolumeclaim
+	oc delete report cm-openshift-persistentvolumeclaim-lookback
+	oc delete report cm-openshift-usage
+	oc delete report cm-openshift-usage-lookback
+	oc delete reportquery cm-node-labels-raw
+	oc delete reportquery cm-openshift-node-labels
+	oc delete reportquery cm-openshift-node-labels-lookback
+	oc delete reportquery cm-openshift-persistentvolumeclaim
+	oc delete reportquery cm-openshift-persistentvolumeclaim-lookback
+	oc delete reportquery cm-openshift-usage
+	oc delete reportquery cm-openshift-usage-lookback
+	oc delete reportquery cm-persistentvolume-labels-raw
+	oc delete reportquery cm-persistentvolumeclaim-labels-raw
+	oc delete reportquery cm-pod-cpu-limit-raw
+	oc delete reportquery cm-pod-labels-raw
+	oc delete reportquery cm-pod-memory-limit-raw
+	oc delete reportquery cm-pod-persistentvolumeclaim-info-raw
```

Wouldn't it be simpler to just reference the directory so we don't have to keep up with a growing list?
```suggestion
	oc delete -f roles/setup/files
```

---

Yes! That is much better. Committed!